src/InnoSetup/Languages/IdfToolsSetup_pt-BR.islu

```diff
@@ -0,0 +1,52 @@
+; Copyright 2019-2020 Espressif Systems (Shanghai) CO LTD
+; SPDX-License-Identifier: Apache-2.0
+
+[LangOptions]
+LanguageName=Portuguese
+LanguageID=$0416
+
+[CustomMessages]
+PreInstallationCheckTitle=Verificação do sistema de pré instalação
+PreInstallationCheckSubtitle=Verificação do ambiente
+SystemCheckStart=Iniciando verificação do sistema ...
+SystemCheckForDefender=Verificação do Windows Defender
+SystemCheckHint=Dica
+SystemCheckResultFound=ENCONTRADO
+SystemCheckResultNotFound=NÃO ENCONTRADO
+SystemCheckResultOk=OK
+SystemCheckResultFail=FALHOU
+SystemCheckResultError=ERR
+SystemCheckResultWarn=WARN
+SystemCheckStopped=Checagem interrompida.
+SystemCheckStopButtonCaption=Parar
+SystemCheckComplete=Checagem completa.
+SystemCheckForComponent=Checando instalação
+SystemCheckUnableToExecute=Não foi possível executar
+SystemCheckUnableToFindFile=Arquivo não encontrado
+SystemCheckRemedyMissingPip=Por favor use uma versão suportada do Python no próximo passo.
+SystemCheckRemedyMissingVirtualenv=Por favor instale o virtualenv e tente novamenta a instalação. Comandos sugeridos:
+SystemCheckRemedyCreateVirtualenv=Por favor use a versão suportada do Python no próximo passo.
+SystemCheckRemedyPythonInVirtualenv=Por favor use a versão suportada do Python no próximo passo.
+SystemCheckRemedyBinaryPythonWheel=Por favor use a versão suportada do Python no próximo passo.
+SystemCheckRemedyFailedHttpsDownload=Por favor use a versão suportada do Python no próximo passo.
+SystemCheckRemedyFailedSubmoduleRun=O Python contém um módulo subprocess.run utilizado pelo Python 2. Por favor remova este módulo. Comandos sugeridos:
+SystemCheckApplyFixesButtonCaption=Aplicar Correções
+SystemCheckFullLogButtonCaption=Log completo
+SystemCheckApplyFixesConsent=Você quer aplicar os comandos com as correções sugeridas, a fim de atualizar o seu ambiente Windows e iniciar uma nova checagem do sistema?
+SystemCheckFixesSuccessful=Correções aplicadas com sucesso.
+SystemCheckFixesFailed=Aplicação das correções falhou. Por favor veja o log completo para maiores informações.
+SystemCheckNotCompleteConsent=Checagem do sistema não concluída. Deseja prosseguir sem a checagem do sistema?
+SystemCheckRootCertificates=Verificando certificados
+SystemCheckRootCertificateWarning=Não foi possível carregar dados do servidor dl.espressif.com.
+SystemCheckForLongPathsEnabled=Verificando registro "LongPathsEnabled" no sistema
+SystemCheckRemedyFailedLongPathsEnabled=Por favor alterar o registro HKLM\SYSTEM\CurrentControlSet\Control\FileSystem\LongPathsEnabled para 1. The operation requires Administrator privileges. Command:
```

Comment:
The operation requires Administrator privileges -> A operação requer privilégios de Administrador

Reply:
Done!